### ISSUE TITLE
Event tables have broken aria-labels

### ISSUE THREAD

**Issue by MinnDevelopment** [keycloak/keycloak#31545]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

We recently tried to update to keycloak 25.0.2 and ran into the problem that all column headers in `/admin/master/console/#/[realmName]/events` now have the same aria label, seemingly copied from the table name.

In **Admin Events** all of them have the label "Admin events" and in **User Events** they are "Events". This breaks our playwright scripts for automated testing, since it overwrites the name of these nodes to a less accessible name.

![image](https://github.com/user-attachments/assets/4df0f8c1-ff7f-4302-8dbc-cd1d5014258a)

![image](https://github.com/user-attachments/assets/98e1b693-4726-4844-bd93-d5c0c146e3e1)

### Version

25.0.2

### Regression

- [X] The issue is a regression

### Expected behavior

Either there should not be aria-labels on these th elements, or they should be more useful and context aware.

### Actual behavior

Labels that overwrite the accessible name of the column headers with a redundant name.

### How to Reproduce?

Open the events page and inspect using any accessibility / devtools in the browser.

### Anything else?

Seems to have been introduced by recent PR that changed these tables: #29812 

Specifically, this line seems to be wrong: https://github.com/keycloak/keycloak/blob/913a2aa07195b6545d2bc27176b97a1216d0f8e9/js/apps/admin-ui/src/components/table-toolbar/KeycloakDataTable.tsx#L153

I'm not entirely sure what the intention here is, for now I would likely just remove these labels entirely.
